core/server/lib/members/users.js

```diff
@@ -10,7 +10,7 @@ module.exports = function ({
     decodeToken
 }) {
     function requestPasswordReset({email}) {
-        return getMember({email}, {require: true}).then((member) => {
```

Comment:
Did we move `require:true` to service layer or just removed it completely for now? 😅

Reply:
It's been moved in the `getMember` function in the members service :relaxed: